src/main/lrsql/util/oidc.clj

```diff
@@ -1,19 +1,187 @@
 (ns lrsql.util.oidc
   "OpenID Connect Utilities"
-  (:require [clojure.spec.alpha :as s]
+  (:require [clojure.core.memoize :as mem]
+            [clojure.java.io :as io]
+            [clojure.spec.alpha :as s]
+            [clojure.string :as cs]
+            [clojure.tools.logging :as log]
+            [com.yetanalytics.lrs.auth :as lrs-auth]
             [com.yetanalytics.pedestal-oidc.discovery :as disco]
-            [lrsql.spec.config :as config]))
+            [com.yetanalytics.pedestal-oidc.interceptor :as oidc-i]
+            [com.yetanalytics.pedestal-oidc.jwt :as jwt]
+            [io.pedestal.interceptor :as i]
+            [lrsql.init.authority :as authority]
+            [lrsql.spec.config :as config]
+            [lrsql.spec.oidc :as oidc]
+            [lrsql.util.auth :as auth]
+            [selmer.parser :as selm-parser]
+            [xapi-schema.spec :as xs])
```

Comment:
`xapi-schema.spec` is an unused require.

Reply:
Nope, the `:statement/authority` key is defined there